plugins/multiserver/com_osvr_Multiserver_Sensics_zSight.json

```diff
@@ -0,0 +1,18 @@
+{
+  "deviceVendor": "Sensics",
+  "deviceName": "Sensics zSight HMD",
+  "author": "Virendra Singh",
+  "version": 1,
+  "lastModified": "2015-09-28",
+  "interfaces": {
+    "tracker": {
+      "count": 1,
+      "bounded": true,
+      "position": false,
+      "orientation": true
+    }
+  },
+  "semantic": {
+    "untared_orientation": "tracker/0"
```

Comment:
Is this usable as the head orientation? (and is it rotated into the right coordinate system?) If so, convention so far has been to call it `hmd` here, and to below (sibling to `"semantic"`) add an `"automaticAliases"` section like this: https://github.com/OSVR/OSVR-Core/blob/master/plugins/multiserver/com_osvr_Multiserver_OSVRHackerDevKit.json#L47

That way it can mostly auto-configure things.